src/main/scala/craft/JTAG.scala

```diff
@@ -20,5 +20,6 @@ trait JTAGTestLevelIO {
   val tdi        = Input(Bool())
   val tdo        = Output(Bool())
   val tclk       = Input(Bool())
+  val tdo_driven = Output(Bool())
```

Comment:
Rather than doing this, I should have a variable in the test driver that checks if tdo is Z. Do you know how to do this @stevobailey?